app/javascript/layouts/student/StudentTopNav.re

```diff
@@ -3,14 +3,18 @@
 let str = React.string;
 
 open StudentTopNav__Types;
+let targetBlank(navTitle) = { navTitle === "Admin" || navTitle === "Dashboard" }
```

Comment:
Hi @lynnbright . This line could be changed to 
`let targetBlank = navTitle => navTitle === "Admin" || navTitle === "Dashboard";`.

Reply:
@harigopal Thanks for your reminder!
After checking, I did miss installing reason-cli and reason-vscode plugin.
`refmt` can work in my local environment now.